src/exposurelog/routers/add_message.py

```diff
@@ -162,7 +208,8 @@ def get_day_obs_from_registries(
                 registry.queryDimensionRecords("exposure", where=query_str)
             )
             if records:
-                return records[0].day_obs
+                print(f"*** {len(records)=}; {type(records[0])=}")
```

Comment:
not a log message?

Reply:
More temporary mypy diagnostics. I'll delete.